File: Proxy/c#/WorldDns.cs
using System.Collections.Generic;
using System.Linq;

namespace Proxy
{
    public class WorldDns
    {
        private Dictionary<string, string> Addresses { get; set; }
        = new Dictionary<string, string>
        {
            { "example.com","1.0.2.4"},
            {"test.com","2.4.3.2"},
            {"anothersite.net","3.4.2.1"}
        };

        public string GetAddress(string domain)
        {
            if (!Addresses.Any(_ => _.Key == domain))
            {
                return "Domain Not Found!";
            }
            else
            {
                return Addresses.First(_ => _.Key == domain).Value;
            }
        }
    }
}

File: Proxy/python/worlddns.py
class WorldDns(): #The original dns class
    def __init__(self) -> None:
        self.addresses = {
                "example.com":"1.0.2.4",
                "test.com":"2.4.3.2",
                "anothersite.net":"3.4.2.1"
            }
    
    def get_address(self,domain:str):
        if self.addresses.get(domain) is not None:
            return self.addresses[domain]
        else:
            return "Domain NotFound!"